code/parachain/frame/cosmwasm/src/ibc.rs

```diff
@@ -159,11 +159,17 @@ impl<T: Config> Pallet<T> {
 	}
 }
 
-#[derive(Default, RuntimeDebug, Eq, PartialEq, Clone)]
+#[derive(RuntimeDebug, Eq, PartialEq, Clone)]
 pub struct Router<T: Config> {
 	_marker: PhantomData<T>,
 }
 
+impl<T: Config> Default for Router<T> {
```

Comment:
why not deriving Default?

Reply:
as I recall runtme macro does not works with attributes.